recipes/serial-monitor-rust/recipe.yaml

```diff
@@ -0,0 +1,61 @@
+context:
+  version: "0.3.2"
+
+package:
+  name: serial-monitor-rust
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/hacknus/serial-monitor-rust/archive/refs/tags/v${{ version }}.tar.gz
+    sha256: 7d9e7f19f3dc86f74852dd0129dcb31feb08a3411269d337dba3463a4f479f01
+
+build:
+  # build numbers can be set arbitrarily
+  number: 0
+  script:
+    - rustc --version
+    - cargo --version
+    - if: osx
+      then:
+        - export MACOSX_DEPLOYMENT_TARGET=10.15  # or the correct macOS version
+        - export PATH="/usr/bin:$PATH"  # Ensure that system Clang is used, not a potentially conflicting Homebrew version
+        - cargo-bundle-licenses --format yaml --output ${SRC_DIR}/THIRDPARTY.yml
+        - cargo install --locked --bins --root ${PREFIX} --path .
+    - if: linux
+      then:
+        - conda install conda-forge::libclang conda-forge::gtk3 conda-forge::libxcb conda-forge::libxkbcommon conda-forge::openssl conda-forge::libudev -y
```

Comment:
you can require these in the `host` section in the recipe. It's basically same as installing them here.